src/components/KLabel/KLabel.vue

```diff
@@ -75,23 +60,30 @@ const hasTooltip = computed((): boolean => !!(props.info || props.help || slots.
   line-height: var(--kui-line-height-30, $kui-line-height-30);
   margin-bottom: var(--kui-space-40, $kui-space-40);
 
-  .is-required {
-    color: var(--kui-color-text, $kui-color-text);
-    font-size: var(--kui-font-size-30, $kui-font-size-30);
-    font-weight: var(--kui-font-weight-semibold, $kui-font-weight-semibold);
-    margin-left: var(--kui-space-20, $kui-space-20);
+  &.is-required {
+    margin-left: var(--kui-space-60, $kui-space-60);
+    position: relative;
+
+    &::before {
```

Comment:
nit/question: sometimes we're using `&:before` and other times `&::before` -- which do we want?

https://css-tricks.com/to-double-colon-or-not-do-double-colon/#:~:text=Note%20the%20double%2Dcolon%20%3A%3A,you%20should%20automatically%20use%20it.

Reply:
Vue 3 doesn't even work on IE 11 so not an issue there